media-gfx/tuxpaint-stamps/tuxpaint-stamps-20220604.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=${PN}-${PV:0:4}.${PV:4:2}.${PV:6:2}
+
+DESCRIPTION="Set of 'Rubber Stamp' images which can be used within Tux Paint"
+HOMEPAGE="http://www.tuxpaint.org/"
```

Comment:
`https` here. Isn't there a secion in that site that's more accurate to for package?